addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1738,6 +1811,57 @@ var SnippetsMenu = Widget.extend({
             $snippet.toggleClass('o_disabled', !check);
         });
     },
+    /**
+     * Gets $element css animations and transitions max duration.
+     *
+     * @private
+     * @returns {Number} max duration (Integer) of the animations and transitions for the overlay target
+     */
+    _getElementMaxAnimationAndTransitionDuration($element) {
+        const cssDurationProperties = ['animation-duration', 'transition-duration'];
+        let maxDuration = 0;
+        for (let index = 0, len = cssDurationProperties.length; index < len; index++) {
+            const key = cssDurationProperties[index];
+            const compStyle = window.getComputedStyle($element.get(0), null);
+            const durationProperty = compStyle.getPropertyValue(key);
+            // Safari returns null and typeof object, but Chrome returns 0s
+            if (durationProperty && durationProperty !== '0s') {
+                const transItems = durationProperty.toLowerCase().split(',');
+                // loop through transition values
+                for (let j = 0; j < transItems.length; j++) {
+                    let duration = transItems[j];
+                    if (duration.indexOf('ms') !== -1) {
+                        // time value in milliseconds
+                        duration = duration.replace('ms', '');
+                        duration = Math.ceil(parseFloat(duration));
+                    } else {
+                        // time value in seconds, convert to milliseconds
+                        duration = duration.replace('s', '');
+                        duration = Math.ceil(parseFloat(duration) * 1000);
+                    }
```

Comment:
There are functions in web_editor.utils to handle unit conversions

Reply:
👍  utils.convertValueToUnit(...)